shell/platform/tizen/flutter_project_bundle.cc

```diff
@@ -23,9 +25,15 @@ namespace {
 // Returns the path of the directory containing the app binary, or an empty
 // string if the directory cannot be found.
 std::filesystem::path GetExecutableDirectory() {
-  auto res_path = app_get_resource_path();
-  if (res_path == nullptr) {
-    return std::filesystem::path();
+  auto* res_path = app_get_resource_path();
+  if (!res_path) {
+    char buffer[PATH_MAX + 1];
+    auto length = readlink("/proc/self/exe", buffer, sizeof(buffer));
```

Comment:
app_get_resource_path always returns absolute path, does `readlink` have the same behavior?